battalion/app/screen/FourDigitCodeInsertScreen.js

```diff
@@ -43,30 +41,32 @@ export default function FourDigitCodeInsertScreen({ route }) {
   };
 
   const handleConfirm = async () => {
-    /*  const combinedSerialNum = combinedSerialNumRef.current;  */
-
     const fourDigitCode = digitValues.join("");
     if (fourDigitCode.length !== 4) {
       console.log("Enter valid digit");
       return;
     }
 
-    // Update the Firestore document with the new fourDigitCode
+    // Update the context with the new password
+    setDevicePassword(fourDigitCode);
```

Comment:
@sup25 the password here should be array of digits, for example [1, 2, 3, 4]

Reply:
Okay